### PR TITLE
Dc swift generator

### PR DESCRIPTION
**Pull request by chhaylatte**

### CODE REVIEW

**Comment by chhaylatte** [comment]
There's a bug in key path for components.  Should be components.componentname.propertypath

File: OriginateTheme/Swift/Models/Fonts.swift [hunk 1]
```diff
@@ -12,7 +12,7 @@ struct Fonts {
     var dictionary: ThemeDefinition
 
     public var body: UIFont {
-        return UIFont.font(keyPath: "fonts.text", dictionary: dictionary, fallback: UIFont.systemFont(ofSize: 16.0))
+        return UIFont.font("fonts.text", dictionary: dictionary, fallback: UIFont.systemFont(ofSize: 16.0))
```
Comment (rweindl) [inline OriginateTheme/Swift/Models/Fonts.swift]: Actually this was only an example. We don't need this line any longer n the final product.

File: OriginateTheme/Swift/Models/Components.swift [hunk 2]
```diff
@@ -8,9 +8,53 @@
...
+    
+    public init(dictionary: ThemeDefinition = [:]) {
+        self.dictionary = dictionary
+    }
+}
+
+public struct NavigationBar {
+    var dictionary: ThemeDefinition
+    
+    public var description: UIFont {
```
Comment (rweindl) [inline OriginateTheme/Swift/Models/Components.swift]: Also here when we merge this PR we shouldn't add those examples here. Let's put them on the README

**Comment by pkluz** [comment]
Sidenote: This PR should also not end up on master straight away, instead let's have it point to `swift-3-support`

**Pull request merged**